server/database/migrations/20200420012727-create-notification.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable("Notifications", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      productId: {
+        type: Sequelize.STRING,
+      },
+      userId: {
```

Comment:
Okay, so I see what you're doing with the userId and dismissed attributes being in this table, but it'll lead to a lot of redundant data in the database. Ideally, there would be 2 tables, one with attributes Discounts:<productId, previousPrice, currentPrice> and another with for Notifications:<productId, userId, dismissed>, and then have onDelete: CASCADE for the first table, so when the sale ends, it automatically ends for the user too.

That way, the same products, with the same productIds, previousPrice and currentPrice wouldn't be repeated in the table for every user who has that product with the same productId, previousPrice and currentPrice in their wishlist(s).

(That last part with the repetition was a little joke)